pkg/doozer/config/deliverable_metadata.go

```diff
@@ -19,13 +19,9 @@ type MetadataDocker struct {
 	Labels     map[string]string `json:"labels"`
 }
 
-// TODO: Consider if "destPath" is available for fetching from base image in some way and can be optional
 type MetadataDoozer struct {
-	SrcPath      string `json:"srcPath"`
```

Comment:
Ved å fjerne denne, blir det da implisitt alle filer i leveransepakken?

Reply:
Det stemmer. Vil det bli et problem ? Jeg tenker at vi styrer hvordan en leveransepakke for doozer blir seende ut